cypress/e2e/environment.cy.js

```diff
@@ -0,0 +1,389 @@
+    //Fill The form and submit
+    cy.visit("/console/");
+    cy.get('[aria-label="Environment card"]').contains("test").click();
+    cy.get('[aria-label="ServiceCatalog-Success"]').should("to.be.visible");
+
+    //Go to settings
+    cy.get('[aria-label="Settings actions"]').click();
+    cy.url().should("contain", "/console/settings?env=");
+    cy.get('[aria-label="Name-value"]').should("contain", "test");
+
+    //Cancel Clear Env and expect nothing to change
+    cy.get("button").contains("Clear environment").click();
+    cy.get("button").contains("Cancel").click();
+    cy.visit("/console/");
+    cy.get('[aria-label="Environment card"]').contains("test").click();
+    cy.get('[aria-label="ServiceCatalog-Success"]').should("to.be.visible");
+
+    //Go to settings
+    cy.get('[aria-label="Settings actions"]').click();
+    cy.url().should("contain", "/console/settings?env=");
+    cy.get('[aria-label="Name-value"]').should("contain", "test");
+
+    //Clear Env
+    cy.get("button").contains("Clear environment").click();
+    cy.get('[aria-label="clear environment check"]').type("test");
+    cy.get("button")
+      .contains("I understand the consequences, clear this environment")
+      .click();
+    cy.visit("/console/");
+    cy.get('[aria-label="Environment card"]').contains("test").click();
+    cy.get('[aria-label="ServiceCatalog-Empty"]').should("to.be.visible");
+    //Update service catalog to restore instances
+    cy.get("button").contains("Update Service Catalog").click();
+    cy.get("button").contains("Yes").click();
+    cy.intercept(
+      "/api/v2/environment/a5f9f7f2-5f43-4fad-a16d-f71f127f0e91?details=fals"
+    ).as("catalogUpdate");
+    cy.wait("@catalogUpdate")
+      .its("response.body")
+      .then(() => {
+        cy.get('[aria-label="ServiceCatalog-Success"]').should("to.be.visible");
+      });
+  });
+
+  it("1.6 Edit environment configuration", function () {
+    cy.visit("/console/environment/create");
+    createEnv(6, true, true);
+    cy.get("button").contains("Submit").click();
+    cy.url().should("contain", "/console/lsm/catalog?env=");
+
+    cy.get('[aria-label="Settings actions"]').click();
+    cy.get('[aria-label="Name-value"]').should("eq", testName(6));
+    cy.get('[aria-label="Settings actions"]').click();
+    cy.get("button").contains("Configuration");
+    cy.wrap(configurationTypes).each((config) => {
+      switch (config.type) {
+        case "text":
+          cy.get("Row-" + config.name + " > td > input")
+            .clear()
+            .tpye(config.newValue);
+          break;
+        case "switch":
+          cy.get("Row-" + config.name + " > td > input").click();
+
+          break;
+        case "select":
+          cy.get("Row-" + config.name + " > td > input").click();
+          cy.get("Row-" + config.name + " > td > .pf-c-select__menu-item")
+            .contains(config.newValue)
+            .click();
+
+          break;
+        case "selecMap":
```

Comment:
selectMap, I suppose? Seems like you miss a 't' in the word

Reply:
Yes, That's the part that is still in progress, but good point, less chance that I'll forget about that :D